src/main/java/core/mate/academy/model/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.mate.academy.model;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
```suggestion
        return Colors.values()[index].name();
```
to get enum String - use name()